README.md

```diff
@@ -1,88 +1,130 @@
+
+3. Paste the token you generated in the earlier step in the `ViewController.swift`.
+
+<img width="700px" src="images/quickstart/xcode-video-quickstart-token.png"/>
 
-> NOTE: If you've used [Twilio's Cocoapods Spec Repo](https://github.com/twilio/cocoapod-specs) before you may need to update it by running `pod repo update twilio` before installing dependencies.
 
-## Access Tokens and Servers
+4. Run the Quickstart app on your iOS device or iOS Simulator. 
 
-Using Twilio's Video client within your applications requires an access token. These access tokens can come from one of two places:
+<img width="500px" src="images/emulator/emulator-iOS-video-quickstart.png"/>
 
-1) You can create a one-time use access token for testing in the [Twilio Console](https://www.twilio.com/console/video/profiles). This access token can be hard-coded directly into your mobile app, and you won't need to run your own server. This works for local development and testing, but isn't a solution for a production environment.
+5. As in Step 2, generate a new Token for another identity (such as "Bob"). Copy and paste the access token into `ViewController.swift` (replacing the one you used earlier). Build and run the app on a second physical device if you have one, or the iPhone simulator.
 
-2) When you're ready to take the next step, you can run your own server that provides access tokens using your Twilio credentials. This server can either run locally on your development machine, or it can be installed on a server. If you run the server on your local machine, you can use [ngrok](https://ngrok.com/) to give the server an externally accessible web address. That way, you can run the quick start app on an actual device, instead of the iOS Simulator.
+6. Once you have both apps running, enter an identical Room name (such as "MyRoom") into both apps, and tap "Connect" to connect to a video Room (you'll be prompted for mic and camera access on the physical device). Once you've connected from both devices, you should see video! (Since the iPhone Simulator doesn't have a camera you'll only see video in one direction if you're using that environment.)
 
-### Generating an Access Token
 
-The first step is to [Generate an Access Token](https://www.twilio.com/user/account/video/dev-tools/testing-tools) from the Twilio developer console. Use whatever clever username you would like for the identity. You will get an access token that you can copy and paste into ViewController.swift.
 
-### Setting up a Video Chat Web Server
+## Examples
+
+You will also find additional examples that provide more advanced use cases of the Video SDK. The currently included examples are as follows:
+
+- [Screen Capturer](ScreenCapturerExample) - Shows how to use `TVIScreenCapturer` to capture the contents of a `UIView`, and how a custom `TVIVideoCapturer` can be implemented to do the same.
+- [Video CallKit](VideoCallKitExample) - Shows how to use Twilio Video with the iOS CallKit framework.
```

Comment:
Lets add a URL iOS CallKit framework -

Shows how to use Twilio Video with the [iOS CallKit framework](https://developer.apple.com/reference/callkit).